how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Quick Start with TiDB
+summary: Learn how to quick start your first TiDB cluster and try out the features.
+category: how-to
+---
+
+# Quick Start with TiDB
+
+This document describes how to try out TiDB, a distributed SQL database. You can choose any of the following three methods to start your first TiDB cluster:
```

Comment:
start your first TiDB cluster -> get started with the TiDB database platform.

Reply:
I replaced the statement of the TiDB cluster to be the TiDB database platform because:
1. to avoid the confusion as I mentioned earlier.
2. TiDB WASM does not start a TiDB cluster. It is only one TiDB server instance in a browser. It does not have PD or TiKV in it.